src/Microsoft.Health.SqlServer/Features/Schema/Messages/Notifications/SchemaUpgradedNotification.cs

```diff
@@ -10,13 +10,16 @@ namespace Microsoft.Health.SqlServer.Features.Schema.Messages.Notifications
 {
     public class SchemaUpgradedNotification : INotification
     {
-        public SchemaUpgradedNotification(int version)
+        public SchemaUpgradedNotification(int version, bool isFullSchemaSnapshot)
         {
             EnsureArg.IsGte(version, 1);
 
             Version = version;
+            IsFullSchemaSnapshot = isFullSchemaSnapshot;
```

Comment:
I understand Robin requested you to make this change, but general practise lets keep these commits separate. Unless urgent. Now I have no context of this change and cannot approve it. Commits should be has small has possible

Reply:
Just for clarity:

[This commit](https://github.com/microsoft/healthcare-shared-components/commit/6e0eb86d88a6ec261cb43f4de841039f764274cb) to master (PR #92) was accidentally reversed in [this commit](https://github.com/microsoft/healthcare-shared-components/commit/6b0337cf49c56c8d49cb52852544268f14b09b2c) to master. The changes are being re-added in this PR. 

I agree that it makes sense to keep PRs small (and I apologize, I did not help by asking Ali to make a bug fix!) - perhaps in the future if a commit is accidentally reversed, we could have a separate PR dedicated to re-adding it to master. @v-shaaal @smithago let me know your thoughts

Reply:
I understand. I will undo Robin's changes here and send out a different PR with her changes that I had mistakenly undone. I will request her to review that PR. This PR will then be specific to fixing the naming conventions and honoring cancellation tokens.

Reply:
Sure, that sounds good to me. Thanks @v-shaaal!

Reply:
Separated that code out from this pull request. New pull request with changes related to schema upgraded notification: https://github.com/microsoft/healthcare-shared-components/pull/107.